app/assets/stylesheets/views/_landing_page.scss

```diff
@@ -22,3 +22,23 @@
     text-decoration: none;
   }
 }
+
+// Consistent lower margins for top-level blocks
+// Self-chaining the class name gives higher
+// specificty and avoids the use of !important.
+//
+// Note the desktop size is hardcoded here as the
+// govuk-spacing(x) helper does not include large
+// enough sizes.
+
+.app-margin-bottom-responsive.app-margin-bottom-responsive  {
+  margin-bottom: govuk-spacing(6);
+
+  @include govuk-media-query($from: tablet) {
+    margin-bottom: govuk-spacing(8);
+  }
+
+  @include govuk-media-query($from: desktop) {
+    margin-bottom: 80px;
```

Comment:
Are you sure this figure is right? Didn't look like that much to me on the PDF. 

Also, have you looked at using the responsive rather than the static spacing from the design system - ideally we would let that do the hard work rather than hand coding the break points ourselves.

Reply:
I eyeballed it based on the Figma design, which doesn't seem to provide a way of measuring it exactly. I can find out the exact figure and change it as necessary.

I did look at the responsive scale, but it doesn't provide numbers that match the design. If we want to use the responsive spacing then we'd need the design to be changed to match it.

Reply:
Worth a chat with Jeremy to get his thoughts on both getting an accurate figure and seeing if it can align with the DS scales, even if the answer is no.